README.md

```diff
@@ -1,37 +1,172 @@
-# platformsh/config-reader
-[![Build Status](https://travis-ci.org/platformsh/platformsh-config-reader-php.svg?branch=master)](https://travis-ci.org/platformsh/platformsh-config-reader-php)
+# Platform.sh Config Reader (PHP)
 
-A small helper to access a Platform.sh application's configuration, via
-[environment variables](https://docs.platform.sh/development/variables.html).
+This library provides a simpler, more streamlined way to interact the a Platform.sh environment.  It defines offers utility methods to access routes and relationships more cleanly than reading the raw environment variables yourself.
```

Comment:
> This library provides a simpler, more streamlined way to interact the a Platform.sh environment.

Simpler and more streamlined than what? Probably should just be "streamlined and easy to use" or something to that effect